iofog-agent-daemon/src/main/java/org/eclipse/iofog/process_manager/ProcessManager.java

```diff
@@ -120,6 +120,18 @@ public void update() {
 		}
 	};
 
+	public void updateMicroserviceStatus() {
+		microserviceManager.getCurrentMicroservices().stream()
+				.forEach(microservice -> {
+					Optional<Container> containerOptional = docker.getContainer(microservice.getMicroserviceUuid());
+					String containerId = containerOptional.get().getId();
+					MicroserviceStatus status = docker.getMicroserviceStatus(containerId);
+					StatusReporter.setProcessManagerStatus().setMicroservicesStatus(microservice.getMicroserviceUuid(), status);
+				});
+		logDebug("Updated microservice status");
```

Comment:
Make this something like "Updated status for @microservice to $status" so that it's useful in the logs.

Reply:
This has been updated